### DISCUSSION TITLE
Black Ops 2 style HUD texture pack (Perks, powerups, more)

### DISCUSSION THREAD

**Discussion by acronmd** [nzp-team/nzportable#759] [Other Content Releases]
I've made a Black Ops 2 Zombies styled texture pack that replaces 

- Perk icons
- Powerup icons
- Grenade and Bouncing Betty icons
- Scorebar / Moneyback graphic
- Bleeding graphic
- Tally marks and round numbers
- (and on certain versions, score text)

You can drop the contents of the 'hud' folder in -> **'nzp/gfx/hud'**
On any versions of nzp with a charset.tga file in -> **'nzp/gfx'** you can replace it with the charset.tga provided
The PSVita version doesn't support transparency so by default I have a dithered moneyback.tga file, but for versions that do support it I've provided a file in the 'transparent_moneyback' folder
The 3DS doesn't support the bigger moneyback file so I've provided a smaller version in the '3ds_moneyback' folder
Tested on PSVita and partly on New 3DS

Download [here (gfx.zip)](https://github.com/user-attachments/files/15509694/gfx.zip) and view previews below

![2024-05-31-005333](https://github.com/nzp-team/nzportable/assets/78101120/28cd525a-e213-44c1-95f7-aaba3e177525)

![2024-05-31-004409](https://github.com/nzp-team/nzportable/assets/78101120/6003ecc0-414d-4e5d-b0c9-16e6850c51ce)

![2024-05-31-004138](https://github.com/nzp-team/nzportable/assets/78101120/12e0f991-7f89-4059-8de8-83ab729a2bf0)

![2011-01-03_03-09-59 931_top](https://github.com/nzp-team/nzportable/assets/78101120/1a5e7801-5451-4178-9484-d5f4e7145982)

**Comment by ToonLink0001** [comment]
Yo this looks good, how did you do that?

**Comment by InfusedStormblessed** [comment]
I'll test it on PSP for you.

**Comment by hwu7wh-lab** [comment]
Thank you so much I've been looking for this for so loooong
